src/common/app-breakaway.tsx

```diff
@@ -30,9 +30,13 @@ import { EntriesCacheManager } from "./core";
 
 // Define lazy pages
 const ProfileContainer = loadable(() => import("./pages/profile-breakaway"));
+const OldProfileContainer = loadable(() => import("./pages/profile-functional"));
+
 const ProfilePage = (props: any) => <ProfileContainer {...props} />;
 
 const CommunityContainer = loadable(() => import("./pages/community-breakaway"));
+const OldCommunityContainer = loadable(() => import("./pages/community-functional"));
```

Comment:
Same goes for this, should figure out exactly what's wrong, this will make the break-away communities redundant